packages/api/src/scripts/emailSend.js

```diff
@@ -0,0 +1,144 @@
+require('dotenv').config({ path: '.env' });
+const axios = require('axios');
+const fs = require('fs');
+const outputCSV = require('./generatecsv').default.generateEmailCSV;
+// Environment variables
+const { CHES_HOST, CHES_AUTH_URL, CHES_CLIENT_SECRET, CHES_CLIENT_ID } = process.env;
+
+// Five requests per second
+const targetRate = 5;
+
+const config = {
+  headers: {
+    authorization: null,
+    'Content-Type': 'application/json',
+  },
+};
+
+async function authenticateChes() {
+  const params = new URLSearchParams();
+  params.append('grant_type', 'client_credentials');
+
+  const config = {
+    headers: {
+      'Content-Type': 'application/x-www-form-urlencoded',
+    },
+    auth: {
+      username: CHES_CLIENT_ID,
+      password: CHES_CLIENT_SECRET,
+    }, // service client ID and secret
+  };
+  try {
+    const response = await axios.post(CHES_AUTH_URL, params, config);
+    return response.data.access_token;
+  } catch (error) {
+    console.error(error);
+    return error;
+  }
+}
+async function updateToken() {
```

Comment:
Could this be renamed to `getChesToken` or something to be more accurate for what it does?